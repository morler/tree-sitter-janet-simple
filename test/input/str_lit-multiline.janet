"this is the first line
and what is this one?"