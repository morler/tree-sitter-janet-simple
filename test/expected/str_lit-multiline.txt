(source [0, 0] - [1, 22]
  (str_lit [0, 0] - [1, 22]))
